### DISCUSSION TITLE
CoKreat 5.2 Use Cases/Enhancements

### DISCUSSION THREAD

**Discussion by sanyapawah** [Sunbird-coKreat/Community#48] [General Discussions]
5.2 Items
Use Case : Reduce spacing in the table in print preview- [JIRA](https://project-sunbird.atlassian.net/browse/CO-216)
Use Case : Adding input field for each selected medium - [JIRA](https://project-sunbird.atlassian.net/browse/CO-217)

Enhancement: Options labels for MCQ questions in Hindi Medium should be क), ख), ग), घ) - [JIRA](https://project-sunbird.atlassian.net/browse/CO-229)

Enhancement : For Maths VSA, SA and LA questions, no lines are required for the answers. A blank space should be provided. 

Enhancement: Rename sourcing reviewer as "Approver"

Use Case : Print preview available for unapproved [JIRA](https://project-sunbird.atlassian.net/browse/CO-230)

Use case : Within Question Set> Create Question. The 2 options should be Objective Questions and Subjective Question - [JIRA](https://project-sunbird.atlassian.net/browse/CO-236)

Use Case : "Within Question Set> Create Question. The following existing sections should come under Objective Questions:
1. MCQ, 2. Fill in the Blanks (FTB), 3. Match the following (MTF) [JIRA](https://project-sunbird.atlassian.net/browse/CO-242)

Use Case : Add "Comprehension Paragraph" type under Subjective Question Type 
[JIRA](https://project-sunbird.atlassian.net/browse/CO-256)

**Comment by sanyapawah** [comment]
Hi @surendrasinghs @pallakartheekreddy
Please see above the use cases for Prashnavali for your reference

**Reply by arun1205** [reply]
@sanyapawah Are these enhancements given a go-ahead from Design Council

**Reply by sanyapawah** [reply]
@arun1205 can you mention who is in the Design council so that I can confirm ?

**Reply by arun1205** [reply]
@vrayulu and santosh v as far as I know :)

**Comment by surendrasinghs** [comment]
@sanyapawah we should meet and brainstorm on the following

1. Translation / multilingual questions [[CO-217] A separate section to input the question and options should come for each medium selected](https://project-sunbird.atlassian.net/browse/CO-217)
2. [[CO-230] Print preview available for unapproved questions](https://project-sunbird.atlassian.net/browse/CO-230)
3. [[CO-242] 3 subtypes of questions under Objective Question Type : MCQ, Fill in the blanks and Match the following](https://project-sunbird.atlassian.net/browse/CO-242) and [[CO-256] Add question type for “Comprehension Paragraph” under Subjective question type](https://project-sunbird.atlassian.net/browse/CO-256)

I’ve initiated an email with relevant people to schedule this discussion.

**Reply by surendrasinghs** [reply]
Please move the CO-217 to inQuiry JIRA and update the relevant details as discussed.
